app.js

```diff
@@ -74,19 +73,13 @@ finishGameButton.addEventListener('click', () => {
     
     pastGamesArr.push(gameObj);
     // HINT: it will be helpful to keep track of these games as objects with 4 properties, one for each piece of state we're tracking
-
     displayAllGames();
 
-
- 
     // reset the initial state to start with a new form
-
-
     resetGame();
     displayCurrentGameEl(name1, name2, score1, score2);
 });
 
-
 function displayCurrentGameEl(name1, name2, score1, score2) {
```

Comment:
This function has access to global variables. You're tracking state in variables declared globally. Do you need to pass them in as arguments?